Tools/MonoGame.Content.Builder.Editor/Platform/Linux/FileAssociation.Linux.cs

```diff
@@ -0,0 +1,159 @@
+﻿using MonoGame.Tools.Pipeline.Utilities;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+
+namespace MonoGame.Tools.Pipeline
+{
+    public static class FileAssociation
+    {
+        // System directories.
+        private const string applicationDirectory = "/usr/share/applications";
+        private const string iconRootDirectory = "/usr/share/icons/hicolor";
+        private const string iconType = "scalable/mimetypes";
+
+        // Content files.
+        private const string contentFolder = "Content";
+        private const string iconFileName = "monogame.svg";
+        private const string oldMimetypeFileName = "mgcb.xml";
+        private const string newMimetypeFileName = "x-mgcb.xml";
+        private const string desktopFileName = "MGCB Editor.desktop";
```

Comment:
Probably best not to have a space in here. I think `mgcb-editor.desktop` for the filename would be better.

Reply:
Fixed.